narwhals/_polars/expr.py

```diff
@@ -181,11 +181,11 @@ def name(self: Self) -> PolarsExprNameNamespace:
 
 
 class PolarsExprDateTimeNamespace:
-    def __init__(self, expr: PolarsExpr) -> None:
+    def __init__(self: Self, expr: PolarsExpr) -> None:
         self._expr = expr
 
-    def __getattr__(self, attr: str) -> Callable[[Any], PolarsExpr]:
-        def func(*args: Any, **kwargs: Any) -> PolarsExpr:
+    def __getattr__(self: Self, attr: str) -> Any:
+        def func(*args: Any, **kwargs: Any) -> Any:
```

Comment:
are the `Callable` / `PolarsExpr` types not correct?

Reply:
Whops! let me roll back these! For some reason, I started to have issues at a certain point. I will double check